packages/client/src/v2-events/utils/commonUtils.test.ts

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Not sure how much value this unit test suite brings us. Let's remove as these are tested on the legacy side too